lang/cs/Org.Apache.REEF.Common/Telemetry/DriverSystemState.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+// 
+//   http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+using System;
+
+namespace Org.Apache.REEF.Common.Telemetry
+{
+    class DriverSystemState : MetricBase<string>
```

Comment:
Previously, DriverMetrics was a class that only contained a string, so this class is a result of expanding on that to make it a metric of type string, though it would make sense to further expand and let the user decide what metrics the driver should collect. Will change string metric to a more generic name and expand IDriverMetrics.